examples/with-mobx/pages/_app.js

```diff
@@ -21,7 +21,7 @@ class MyMobxApp extends App {
 
   constructor(props) {
     super(props)
-    const isServer = typeof window === 'undefined'
+    const isServer = !process.browser;
```

Comment:
`process.browser` is replaced, meaning it becomes `isServer = false` / `isServer = true` before passing it into Terser (minifier), so it makes more sense to not assign it to a variable as Terser will be able to remove the dead code branch then.